docs/book/orders/returns.rst

```diff
@@ -145,25 +145,25 @@ API coverage
 
 Sylius Plus provides a route that allows accepting or rejecting return request with an API call:
 
-.. code-block:: php
+.. code-block:: text
 
   POST /api/v1/return-requests/{id}/accept
 
 The ``id`` is an id of return request that we want to accept. Content of the request may contain response of return request:
 
-.. code-block:: php
+.. code-block:: json
 
   {
       "response": "Return request confirmed and accepted."
   }
 
-.. code-block:: php
+.. code-block:: test
```

Comment:
this should be text right?

Reply:
Ah, yes. This is probably why the pipeline fails. Thanks.

Reply:
Nope it wasn't.